lib/v2/njglyy/utils/index.js

```diff
@@ -0,0 +1,50 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+
+async function getNoticeList(ctx, url, host, listSelector, itemSelector, titleSelector, contentSelector) {
+    const response = await got({ url, https: { rejectUnauthorized: false } });
```

Comment:
The HTTPS certificate is valid, no need to use `rejectUnauthorized` option

Reply:
> The HTTPS certificate is valid, no need to use `rejectUnauthorized` option

I get it.